users/user-guide-query/multi-stage-query/operator-types/aggregate.md

```diff
@@ -0,0 +1,194 @@
+---
+description: >-
+  Describes the aggregate relation operator in the multi-stage query engine.
+---
+
+# Aggregate relational operator
+
+The aggregate operator is used to perform calculations on a set of rows and return a single row of results.
+
+This page describes the aggregate operator defined in the relational algebra used by multi-stage queries.
+This operator is generated by the multi-stage query engine when you use _aggregate functions_ in a query either with or
+without a `group by` clause.
+
+## Implementation details
+Aggregate operations may be expensive in terms of memory, CPU and network usage. 
+As explained in [understanding stages](../understanding-stages.md), the multi-stage query engine breaks down the query 
+into multiple stages and each stage is then executed in parallel on different _workers_.
+Each worker processes a subset of the data and sends the results to the _coordinator_ which then aggregates the results.
+When possible, the multi-stage query engine will try to apply a divide-and-conquer strategy to reduce the amount of data
+that needs to be processed in the _coordinator_ stage.
+
+For example if the aggregation function is a sum, the engine will try to sum the results of each worker before sending
+the partial result to the _coordinator_, which would then sum the partial results in order to get the final result.
+But some aggregation functions, like `count(distinct)`, cannot be computed in this way and require all the data to be
+processed in the _coordinator_ stage.
+
+In Apache Pinot 1.1.0, the multi-stage query engine always keeps the data in memory.
+This means that the amount of memory used by the engine is proportional to the number of groups generated by the
+`group by` clause and the amount of data that needs to be kept for each group (which depends on the aggregation 
+function).
+
+Even when the aggregation function is a simple `count`, which only requires to keep a long for each group in memory,
+the amount of memory used can be high if the number of groups is high.
+This is why the engine limits the number of groups.
+By default, this limit is 100.000, but this can be changed by providing hints.
```

Comment:
let's standardize on using notation like 100k, so as to not confuse cross geo folks :)

Reply:
Just to be sure: Is there another place where I'm saying it is 100? Or it is a matter of using `.` instead of `,`? If it is the later, just as a curiosity, at least in Spanish speaking countries the `.` is commonly used to mark thousands, millions, etc while `,` is used to separate the integer part and the fractional part ([wiki link](https://en.wikipedia.org/wiki/Decimal_separator#:~:text=In%20English%2Dspeaking%20countries%2C%20the,%2C)%20placed%20on%20the%20baseline.)).

I'm with you that we need to change this to something like 100k that will mean the same for everyone